pod/cut/templates/video_cut.html

```diff
@@ -72,7 +72,7 @@
   <div>
     <span class="float-end">
       <button type="button" class="btn btn-primary btn-sm m-2" data-bs-toggle="modal" data-bs-target="#ConfirmationModal"><i class="bi bi-scissors"></i>&nbsp;{% trans 'Cut' %}</button>
-      <a href="{% url 'video:video' slug=video.slug %}" title="{% trans 'View the video'%}" class="btn btn btn-secondary btn-sm">
+      <a href="{% url 'video:video' slug=video.slug %}" title="{% blocktrans with video_title=video.title %}Back to the video “{{ video_title }}”{% endblocktrans %}" class="btn btn btn-secondary btn-sm">
         <i class="bi bi-film"></i>&nbsp;{% trans 'Back to the video' %}
```

Comment:
idem, le texte du lien et la valeur de son attribut title ne sont pas les mêmes...

Reply:
Tu aurais pu mettre : `Back to the video to view it` ou quelque chose comme ça. Car le titre ne spécifie pas qu'il y un un retour

Reply:
Autant pour moi, j'ai mal regardé...